src/pages/activity/index.tsx

```diff
@@ -7,9 +7,11 @@ import Layout from '../../components/layout/Layout';
  */
 const ActivityPage: NextPage = () => {
   const ActivityView = dynamic(() => import('../../components/activity/ActivityView'));
+  const StatsView = dynamic(() => import('../../components/activity/StatsView'))
   return (
     <Layout>
       <ActivityView />
+      <StatsView />

```

Comment:
Do we want to limit this to just staff/interns?

Reply:
Yeah we probably should, how do you do that?

Reply:
We don't use ADMIN role (I should get rid of it.) This should work. You can add `|| user.role === UserRole.INTERN` if you think interns should see it to
```
{ session.user.role === UserRole.STAFF &&  <StatsView />}
```

Reply:
Ok I kept it as just staff for now